app/models/training/module.rb

```diff
@@ -33,6 +33,12 @@ def self.by_name(name)
       end
     end
 
+    # @return [Boolean]
+    def self.new_module?
+      # check the position as well as the name to handle duplicated mods in the test cms env
+      TrainingModuleRecord.pluck(:name).include?(name) && !TrainingModuleRecord.pluck(:module_position).include?(position)
```

Comment:
I believe in our discussions we identified this as a method that should be replaced

Reply:
Removed [c30aa05](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/747/commits/c30aa056a3eea7ad37f7a051fea48e769a647aa1)